discern.lic

```diff
@@ -0,0 +1,42 @@
+=begin
+  Documentation: https://elanthipedia.play.net/Lich_script_repository#Discern
+=end
+
+custom_require.call(%w[common common-travel events spellmonitor drinfomon])
+
+class AutoDiscern
+  include DRC
+  include DRCA
+
+  def initialize
+    settings = get_settings
+    UserVars.discerns = {} if settings.force_discerns
+    echo 'Reading spell list'
+
+    settings.buff_spells.each do |x|

```

Comment:
I have a similar thought here about method chaining, something like:
```ruby
settings.buff_spells
    .select { |x| x[1]['use_auto_mana'] }
    .each do |x| ...
```

What do you think?